### ISSUE TITLE
Operations tracing troubleshooting guide is incorrect

### ISSUE THREAD

**Issue by mchmarny** [dapr/docs#886]
Our app config docs/examples indicate that the way you turn tracing sample rate is using `samplingRate`

```yaml
spec:
  tracing:
    samplingRate: "1"
```

But our tracing docs/samples show more verbose config which does not include the `samplingRate`

```yaml
spec:
  tracing:
    enabled: true
    exporterType: zipkin
    exporterAddress: "http://localhost:9411/api/v2/spans"
    expandParams: true
    includeBody: true
```

Few questions after reviewing the tracing docs: 

* Is the union of these two what's required? 
* Is `enabled` to turn it on and `samplingRate` to set the rate? 
* What are the defaults `expandParams` and `includeBody`?
* Can `exporterAddress` point to zipkin service in a different namespace?

**Comment by mchmarny** [comment]
I've opened the issue from the `Create documentation issue` in new docs UI. @AaronCrawfis do we have the rights defaults there?

**Comment by mchmarny** [comment]
Hm, when I try it now the issues opened in the above way default to the dapr/docs repo so no idea how this one ended up in dapr/dapr

**Comment by AaronCrawfis** [comment]
https://docs.dapr.io/operations/troubleshooting/setup-tracing/ is the new page

**Comment by tcnghia** [comment]
This is fixed in #995 .  @mchmarny  can you please help close the issue? Thanks
